forge/routes/api/settings.js

```diff
@@ -75,6 +76,7 @@ module.exports = async function (app) {
                     }
                 })
                 response['platform:stats:token'] = app.settings.get('platform:stats:token')
+                response['platform:auth:permissions'] = Permissions
```

Comment:
Given we aren't including custom scopes for this iteration we don't need to expose the full permissions table.